setup.py

```diff
@@ -43,14 +36,21 @@
 ]
 PLATFORMS = "Any"
 INSTALL_REQUIRES = ["numpy", "pandas", "xarray", "netCDF4", "packaging"]
+# Configuration for setuptools-scm
+SETUP_REQUIRES = ["setuptools_scm"]
+USE_SCM_VERSION = {
+    "local_scheme": "node-and-date",
+    "write_to": "_version.py",
```

Comment:
I wonder if writing to a `_version.py` file is actually needed here.

Reply:
https://github.com/fatiando/boule doesn't write the file.

Reply:
Ok, I'll remove this line then (and don't add `_version.py` to .gitignore too).